lighthouse-core/audits/dobetterweb/has-inspector-issues.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+ /**
+ * @fileoverview Audits a page to determine whether it generates issues in the Issues panel of Chrome Devtools.
+ * The audit is meant to maintain parity with the Chrome Devtools Issues panel front end.
+ * https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/sdk/
+ */
+
+'use strict';
+
+/** @typedef {{issueType: string, description: string, requestUrl?: string}} IssueItem */
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on various types of issues with the page. This descriptive title is shown to users when no issues were logged into the Chrome DevTools Issues panel. */
+  title: 'No issues in the `Issues` panel in Chrome Devtools',
+  /** Title of a Lighthouse audit that provides detail on various types of issues with the page. This descriptive title is shown to users when issues are detected and logged into the Chrome DevTools Issues panel. */
+  failureTitle: 'Issues were logged in the `Issues` panel in Chrome Devtools',
+  /** Description of a Lighthouse audit that tells the user why issues being logged to the Chrome DevTools Issues panel are a cause for concern and so should be fixed. This is displayed after a user expands the section to see more. No character length limits. */
+  description: 'Issues logged to the `Issues` panel in Chrome Devtools indicate unresolved problems. ' +
+  'They can come from network request failures, insufficient security controls, ' +
+  ' and other browser concerns. ',
+  /** Table column header for the type of issue. */
+  columnIssueType: 'Issue',
+  /** Message shown in a data table when the item is a SameSiteCookie issue. */
+  sameSiteMessage: 'A cookie\'s [`SameSite`] attribute was not set or is invalid',
```

Comment:
Still not sure if it's worth the trouble to break the SameSite reasons into more specific strings as they have in the DevTools front end. Reason being is there are quite a few different scenarios that would need to be covered compared to the other issues. Also, it's unclear which strings are temporary vs long-term as some strings may be deprecated/removed in the future (see [comment in chromium frontend code](https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/sdk/SameSiteCookieIssue.js;l=518)).